app/models/task.py

```diff
@@ -1,5 +1,27 @@
 from app import db
 
 
+
 class Task(db.Model):
     task_id = db.Column(db.Integer, primary_key=True)
+    title = db.Column(db.String)
+    description = db.Column (db.String)
+    completed_at = db.Column(db.DateTime, nullable=True)
+    goal_id = db.Column(db.Integer, db.ForeignKey('goal.goal_id'),
+        nullable=True)
+
+    def to_dict(self):
+        dictionary = { "id": self.task_id,
```

Comment:
Nitpick: instead of `dictionary` as a variable name, I think that something like `task` would be better.